datajunction-server/datajunction_server/models/node.py

```diff
@@ -505,6 +506,127 @@ def is_available(
         return True
 
 
+class MetricDirection(str, enum.Enum):
+    """
+    The direction of the metric that's considered good, i.e., higher is better
+    """
+
+    HIGHER_IS_BETTER = "higher_is_better"
+    LOWER_IS_BETTER = "lower_is_better"
+    NEUTRAL = "neutral"
+
+
+class MetricKind(str, enum.Enum):
```

Comment:
Have you considered `MetricType` ? Kind sounds weird, but maybe we'll get used to it.

Also, I feel a better the default would be `GENERIC` instead of `UNSPECIFIED`, but its your choice.

Reply:
So I ended up removing `MetricKind` altogether, because it seems like the values captured here could be encapsulated in `MetricUnit` instead (i.e., percentage or proportion). There are also too many values here, making it hard to reason about what this field is meant to be.